services_and_applications/slate/use_cases/gitlab_runner.rst

```diff
@@ -0,0 +1,128 @@
+either the "Developer" or "Administrator" perspective indicated. These two perspectives present two different ways of viewing resources
+deployed to the cluster. The Developer Perspective focuses on a Topology overview for the resources and will be used for the deployment
+of a GitLab Runner.
+
+To deploy a GitLab Runner from the Developer Perspective, navigate to "Topology" -> "Helm Chart" and select
+"GitLab Runner vX.X.X Provided by Slate Helm Charts". From the new window, select "Install Helm Chart".
+
+On the resulting screen, one can customize the installation of the GitLab Runner helm chart. The chart has been forked from upstream to
+allow for customized deployment to Slate. From the Form View, expand the "GitLab Runner" fields. Using the registration token retrieved
+in the prior section, enter token for adding the runner to the GitLab server in the empty field.
+
+.. note::
+
+   If the Registration Token is left blank, the runner installation will not succeed.
+
+If the GitLab Runner needs to be able to use batch scheduler command, toggle the "Enable Batch Scheduler" option to true. Then, select the
+necessary filesystem annotation to use for the cluster the chart is being deployed. For Marble, the "olcf" annotation would be used whereas
+whereas for Onyx the "ccsopen" annotation would be used.
+
+The Resources requests and limits are setup to match the namespace default limits. If more resources are anticipated for the deployment,
+modify the requests and limits as needed.
+
+This helm chart was forked from the the upstream GitLab Runner helm chart. As such, nearly all of the parameters documented in the
+upstream chart could be provided here as well.
+
+To verify that the GitLab runner was properly deployed, one could explore the deployed resources from the Topology view from the
+Slate web console. Or, from the command line:
+
+.. code-block:: bash
+
+   $ helm ls
+   NAME         	NAMESPACE    	REVISION	UPDATED                               	STATUS  	CHART              	APP VERSION
+   gitlab-runner	myproject   	1       	2021-09-15 14:10:15.12187869 +0000 UTC	deployed	gitlab-runner-1.0.0	14.2.0
+   $ oc get pods
+   NAME                                          READY   STATUS    RESTARTS   AGE
+   gitlab-runner-gitlab-runner-687486d94-lpmhs   1/1     Running   0          107s
+
+Once the pod has a Status of "Running", navigate to the GitLab web interface and ensure that the runner has registered to either the group or repository per the token given, and that it is listed as available.
+
+Finally, if batch scheduler integration was enabled, one can verify functionality in the pod with:
+
+.. code-block:: bash
+
+   $ oc exec -it pods/gitlab-runner-gitlab-runner-687486d94-lpmhs -- bash
+   bash-5.0$ squeue
+             JOBID PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON)
+              9951 ...
+
+Removing a GitLab Runner
+^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
For the removing portion, would it make more sense to use the UI since that is how the user installed it? Otherwise we should also go through where to get the `helm` binary since a new users may not have that installed

Reply:
modified to remove via the developer perspective.